client/src/components/imagine25/Survey.js

```diff
@@ -0,0 +1,69 @@
+import { React, useState } from "react";
+import { PropTypes } from "prop-types";
+import QuestionCount from "../quiz/components/QuestionCount";
+import AnswerOption from "./AnswerOption";
+
+function Survey(props) {
+  function renderAnswerOptions(key) {
+    return (
+      <AnswerOption
+        key={key.index}
+        answerContent={key.content}
+        answerType={key.index}
+        answer={props.answer}
+        questionId={props.questionId}
+        questionType={props.questionType}
+        onAnswerSelected={props.onAnswerSelected}
+        onMultiSelected={props.onMultiSelected}
+      />
+    );
+  }
+  return (
+    <div className="quiz container" key={props.questionId}>
+      <QuestionCount counter={props.questionId} total={props.questionTotal} />
+      <h2 className="quiz question">
+        {props.question} {props.multiChoice && " Select all that apply."}
+      </h2>
+      <ul className="answerOptions">
+        {props.answerOptions.map(renderAnswerOptions)}
+      </ul>
+      <div className="align-right">
+        {props.questionId !== props.questionTotal && !props.isUnderAge ? (
+          <button
+            className="btn btn-second text-uppercase  nextButton"
+            onClick={props.nextQuestion}
+            disabled={props.disable}
+          >
+            Next Question
+          </button>
+        ) : (
+          <button
+            className="btn btn-second text-uppercase  nextButton"
+            onClick={props.onComplete}
+            disabled={props.disable}
+          >
+            Complete
```

Comment:
nit: make sure the next question and complete button stay in the frame

![Screenshot 2025-02-10 at 2 00 49 PM](https://github.com/user-attachments/assets/fed7ca71-eb92-43bf-a854-48d5010936ab)